assets/js/src/frontend/blog.js

```diff
@@ -84,9 +97,17 @@ const requestMorePosts = () => {
 	const requestUrl = maybeParseUrlForCustomizer(url);
 	page++;
 
+	canFetchPosts = false;
+	const reset = setTimeout(() => {
+		canFetchPosts = true;
+	}, 1500);
```

Comment:
Instead of using a `setTimeout` what do you think about using a placeholder for each response that we replace once a response is received?

E.g. when a request is emitted we add a DOM element for the response that we later replace with the response or remove it entirely.

Reply:
When a response is made, the spinner is added to the DOM to highlight the loading. Do you suggest that if we make two requests, each request should have its spinner and be replaced in the order they were created? I was thinking of making a sync that reorders the posts based on their triggering, but this will create a lot of layout shifts in the page.

Reply:
I was thinking of having an invisible DOM element for each page and you could replace that with the content or remove it if the response is not valid. These would added before requesting each page, this way the order will be preserved even when the responses finish at different times. This way I think you minimize the layout shifts although there will be cases where you have this but not as much as reordering them.

Another possible solution to consider would be to await the Promise and synchronize the JS and just run each request serialized instead of in parallel. Having a `setTimeout` should be avoided as it is not reliable and does not guarantee de desired output 100% of the time.

Reply:
I used the first option. It was more minimal thanks to a simple trick with `.outerHTML`. The second option might be more correct but it was heavier in size.

Thanks 🙏